[RFC] Limit --coverage flag to files under "src"

tarruda opened the pull request:
Should fix #2632

justinmk commented:
:+1: Much better than what I suggested in #2632. Coveralls seems happy with it. LGTM

fwalch commented:
Can you add "resolves #2632" and a short explanation why this change was made to the commit message?

tarruda commented:
> Can you add "resolves #2632" and a short explanation why this change was made to the commit message?

:+1:

tarruda committed:
build: Limit --coverage flag to files under "src"

Resolves #2632. This is done so C helper modules don't generate unexpected
coverage output.

Reviewed-by: Justin M. Keyes <justinkz@gmail.com>
Reviewed-by: Florian Walch <florian@fwalch.com>
